src/modules/dashboard/components/ColonyDecisionPreview/DetailsWidget/UserWidget/UserWidget.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
I think that the same is true for the UserWidget - leftover from a rebase and should be deleted